pkg/tests/apis/provisioning/exportunifiedtorepository/dashboard-test-v0.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: dashboard.grafana.app/v0alpha1
```

Comment:
Should we assert more than resulting exported files? Basically the entire file if possible. 

I think it would be very valuable as we will be able to see what will come out and test also the removal or addition of things by mutator an whichever dark magic we have in between.

Reply:
I'll add some more -- without this change we export a nil spec.  the v0 vs v1 difference is only notional now -- v1 *should* imply that we expect validation to pass...  actually now that we have this we should be able to remove our fieldValidation=Ignore 🤔